xml/installation.xml

```diff
@@ -29,143 +26,31 @@
  </info>
 
   <sect1 xml:id="sec2-installation-systemrole">
-   <title>System roles for &sles; &hpcm; &productnumber;</title>
+   <title>Installing &sles; &hpcm; &productnumber;</title>
```

Comment:
Same here: Why should the user install the HPC Module?
Start the section with goal, not the procedure: _Installing SLES for HPC_